moveit_core/planning_request_adapter/src/planning_request_adapter.cpp

```diff
@@ -45,22 +45,30 @@ rclcpp::Logger LOGGER = rclcpp::get_logger("moveit").get_child("planning_request
 
 namespace
 {
-bool callPlannerInterfaceSolve(const planning_interface::PlannerManager& planner,
-                               const planning_scene::PlanningSceneConstPtr& planning_scene,
-                               const planning_interface::MotionPlanRequest& req,
-                               planning_interface::MotionPlanResponse& res)
+moveit::core::MoveItErrorCode callPlannerInterfaceSolve(const planning_interface::PlannerManager& planner,
+                                                        const planning_scene::PlanningSceneConstPtr& planning_scene,
+                                                        const planning_interface::MotionPlanRequest& req,
+                                                        planning_interface::MotionPlanResponse& res)
 {
   planning_interface::PlanningContextPtr context = planner.getPlanningContext(planning_scene, req, res.error_code_);
   if (context)
-    return context->solve(res);
+  {
+    // TODO(andyz): consider returning a moveit::core::MoveItErrorCode from context->solve()
+    bool result = context->solve(res);
```

Comment:
@henningkayser I replaced `bool` with `moveit::core::MoveItErrorCode` up to this point in planning_interface::PlanningContext:

`bool solve(MotionPlanDetailedResponse& res)`

If this PR gets merged, I'll make an issue to recommend replacing `bool` even further up the pipeline.